### DISCUSSION TITLE
Empty Event List

### DISCUSSION THREAD

**Discussion by jkyselak** [blakeblackshear/frigate#1080] [Ask A Question] [answered]
I just noticed that the Camera Event List is empty and no longer shows the recorded clips. 
Everything was running smoothly for weeks and I did not make any changes to the configuration.
Could the core-2021.5 update have broken something?
I still see the clips in the media browser but under library, I see two folders, one "Local Media" and one "Frigate".
The clips are only listed under "Local Media" and then "/frigate", "clips/".
Seems the Camera Event List points to the wrong directory.
How can I fix this?
Any help is appreciated

Answering my own question: When I looked at the frigate.db file in the /media/frigate/clips/ directory I saw that the database had been migrated and I guess therefore been reset. Once a new event came in, it showed up in the events list.

**Comment by blakeblackshear** [comment, marked as answer]
The events list pulls from the sqlite database that is stored in the clips directory by default. If that file is deleted, the clips will be retained in the folder, but they won't show up in the UI.
